openid-provider-commands-1_0.md

```diff
@@ -474,11 +502,28 @@ The functionality of the Unauthorize Command is also performed by Suspend, Archi
 
 The RP MUST revoke all active sessions and MUST reverse all authorization that may have been granted to applications, including `offline_access`, for Account resources identified by the `sub`.
 
+## Manage Command
+
+Identified by the `manage` or `manage_async` value in the `command` Claim in a Command Token.
+
+The OP sends this Command to request a change in management responsibility for an Account. The Command Token MUST include the following additional claim:
+
+- **managed_by**  
+  REQUIRED.  
+  A JSON string indicating the requested management state. MUST be set to `OP` if the OP wants to take over management of the Account, or `RP` if the OP wants to hand management back to the RP.
+
+The RP processes the Manage Command based on the current management state of the Account and the requested `managed_by` value:
 
+- If `managed_by` is `OP` and the RP allows the OP to take over management, the RP updates the Account's management status to indicate it is managed by the OP.
+- If `managed_by` is `RP` and the Account is currently managed by the requesting OP, and the RP will take over management, the RP updates the Account's management status to indicate it is managed by the RP.
 
+### Access Denied Error
 
+If the RP does not allow the OP to change management of the Account, the RP MUST return an HTTP 403 Forbidden response and include the `error` parameter with the value of `access_denied`.
```

Comment:
The term "allow" seems very broad as there could be other reasons an operation is not allowed such as the `management_not_transferable` error out have outlined below.  This error feels like it should be scoped to authorization only such as ` RP does not authorize the OP to change management of the Account`